xbmc/addons/AddonManager.cpp

```diff
@@ -564,6 +567,40 @@ bool CAddonMgr::FindInstallableById(const std::string& addonId, AddonPtr& result
   return true;
 }
 
+bool CAddonMgr::GetInstalledBinaryAddons(BINARY_ADDON_LIST& binaryAddonList)
```

Comment:
but yeah if it is too much effort, leave it as it is for now. can be optimized later.